docs/remote_inference_blueprints/standard_blueprints/bedrock_connector_cohere_cohere.embed-english-v3_standard_blueprint.md

```diff
@@ -0,0 +1,190 @@
+# Bedrock connector standard blueprint example for Cohere text embedding model
+This blueprint demonstrates how to deploy a cohere.embed-english-v3 using the Bedrock connector without pre and post processing functions.
+This is recommended for models to use the ML inference processor to handle input/output mapping.
+Note that if using a model that requires pre and post processing functions, you must provide the functions in the blueprint. Please refer to legacy blueprint: [Bedrock connector blueprint example for Cohere embed-english-v3 model](https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/bedrock_connector_cohere_cohere.embed-english-v3_blueprint.md)
+
+## 1. Add connector endpoint to trusted URLs:
+
+Note: no need to do this after 2.11.0
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://bedrock-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+        ]
+    }
+}
+```
```

Comment:
Can we remove altogether? These are blueprints designed for ML processors, which is > 2.11. And, from my experience this adds confusion and downstream issues when testing models from different endpoints anyway

Reply:
sounds good, I will remove the first step by adding endpoints to setting